Crash report: Failed to connect to api.ashcon.app -> invalid memory address or nil pointer dereference

```
ts=2022-02-09T14:41:22.079Z caller=exporter.go:450 level=error msg="Failed to connect to api.ashcon.app" err="Get \"https://api.ashcon.app/mojang/v2/user/0e3d82b5-ddfb-42ad-9cdd-ba39766ed255\": read tcp IP.ADD.RESS.HERE:44562->IP.ADD.RESS.HERE:443: read: connection reset by peer"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x83c8e4]

goroutine 69201 [running]:
github.com/minecraft-exporter/pkg/exporter.(*Exporter).getPlayerStats(0xc000242240, 0x1)
        /home/runner/work/minecraft-prometheus-exporter/minecraft-prometheus-exporter/pkg/exporter/exporter.go:453 +0x564
github.com/minecraft-exporter/pkg/exporter.(*Exporter).Collect(0xc000242240, 0xc0002f6f60)
        /home/runner/work/minecraft-prometheus-exporter/minecraft-prometheus-exporter/pkg/exporter/exporter.go:826 +0x21a
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
        /home/runner/go/pkg/mod/github.com/prometheus/client_golang@v1.12.1/prometheus/registry.go:446 +0x102
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
        /home/runner/go/pkg/mod/github.com/prometheus/client_golang@v1.12.1/prometheus/registry.go:538 +0xb4d
```

--- a/pkg/exporter/exporter.go
+++ b/pkg/exporter/exporter.go
@@ -447,12 +447,12 @@ func (e *Exporter) getPlayerStats(ch chan<- prometheus.Metric) error {
 		case "mojang":
 			resp, err := http.Get(fmt.Sprintf("https://api.ashcon.app/mojang/v2/user/%s", id))
 			if err != nil {
-				level.Error(e.logger).Log("msg", "Failed to connect to api.ashcon.app", "err", err) // nolint: errcheck
+				return level.Error(e.logger).Log("msg", "Failed to connect to api.ashcon.app", "err", err) // nolint: errcheck
 			}
 
 			if resp.StatusCode == 200 {
 				if err := json.NewDecoder(resp.Body).Decode(&player); err != nil {
-					level.Error(e.logger).Log("msg", "Failed to connect decode response", "err", err) // nolint: errcheck
+					return level.Error(e.logger).Log("msg", "Failed to connect decode response", "err", err) // nolint: errcheck
 				}
 			} else {
 				return fmt.Errorf("error retrieving player info from api.ashcon.app: %w", fmt.Errorf(fmt.Sprintf("Status Code: %d", resp.StatusCode)))

--- a/pkg/exporter/exporter.go
+++ b/pkg/exporter/exporter.go
@@ -447,12 +447,12 @@ func (e *Exporter) getPlayerStats(ch chan<- prometheus.Metric) error {
 		case "mojang":
 			resp, err := http.Get(fmt.Sprintf("https://api.ashcon.app/mojang/v2/user/%s", id))
 			if err != nil {
-				return level.Error(e.logger).Log("msg", "Failed to connect to api.ashcon.app", "err", err) // nolint: errcheck
+				return level.Error(e.logger).Log("msg", "Failed to connect to api.ashcon.app", "err", err)
 			}
 
 			if resp.StatusCode == 200 {
 				if err := json.NewDecoder(resp.Body).Decode(&player); err != nil {
-					return level.Error(e.logger).Log("msg", "Failed to connect decode response", "err", err) // nolint: errcheck
+					return level.Error(e.logger).Log("msg", "Failed to connect decode response", "err", err)
 				}
 			} else {
 				return fmt.Errorf("error retrieving player info from api.ashcon.app: %w", fmt.Errorf(fmt.Sprintf("Status Code: %d", resp.StatusCode)))

